config/webpack/webpack.config.static.js

```diff
@@ -29,8 +30,14 @@ module.exports = {
   resolve: base.resolve,
   module: base.module,
   plugins: [
+    new webpack.optimize.DedupePlugin(),
+    new webpack.optimize.UglifyJsPlugin({
```

Comment:
I don't _think_ it makes any difference, but I like to think of the plugin array as a pipeline / middleware stack where order is significant. As such, it makes more sense to first Clean, then Define, then Dedupe, then Uglify.